src/IRrecv.cpp

```diff
@@ -954,6 +954,10 @@ bool IRrecv::decode(decode_results *results, irparams_t *save,
     DPRINTLN("Attempting GOODWEATHER decode");
     if (decodeGoodweather(results, offset)) return true;
 #endif  // DECODE_GOODWEATHER
+#if DECODE_GORENJE
+    DPRINTLN("Attempting GORENJE decode");
+    if (decodeGorenje(results, offset)) return true;
```

Comment:
Please move this to the end, unless there is a specific need to have it early.

Reply:
Sure, will do. 
I thought this list is alphabetical, actually, but I guess I wasn't paying close enough attention.

Reply:
LGTM now.